psi4/src/psi4/psimrcc/operation_compute.cc

```diff
@@ -188,13 +182,6 @@ void CCOperation::tensor_product() {
 }
 
 void CCOperation::contract() {
-    if (compatible_contract() && (reindexing.size() == 0)) {
-        // Same indexing contract, let BLAS directly handle this (although we guide it)
-        DEBUGGING(4, outfile->Printf("\n...same indexing for the target and the output of this operation"););
-    } else {
-        // Different indexing contract, we work this by hand at first
-        DEBUGGING(4, outfile->Printf("\n...different indexing for the target and the output of this operation"););
```

Comment:
should any of these be warnings or exits?

Reply:
My read of the code is that these are **not** warnings or exits. They just indicate whether or not the algorithm needs to account for complicated tensor re-indexing. Which is exactly the kind of thing you'd want to know if you were hunting for a bug in this function.